content/blog/2024-12-03-tmil-11/index.md

```diff
@@ -0,0 +1,146 @@
++++
+title = "Linebender in November 2024: color!"
+authors = ["Bruce Mitchener", "Daniel McNab"]
++++
+
+Linebender is an informal open-source organization working on various projects to advance the state of the art in GUI for [the Rust programming language](https://rust-lang.org).
+
+This month's update contains some surprises, even to us!
+
+## Color
+
+When we published the update for [October, 2024](/blog/tmix-10/), we had already been discussing starting a new color-focused crate, but hadn't yet created a repository.
+
+We may have a dedicated blog post about the Color crate in the future and laying out some of the motivations, but we'll go over it a bit here.
+
+Linebender, among other things, believes in helping build solid foundations for people building user interfaces and working in graphics, in the Rust programming language (and on GPUs).
+Upon seeing a problem, we would rather produce a focused crate that solves that problem and that can be used by others within the Rust ecosystem.
```

Comment:
Language nitpick: the "rather" here is currently hanging. Maybe "we like to produce ...", or maybe you had different ideas about this sentence.

Reply:
Reworded slightly.